CHANGELOG.md

```diff
@@ -7,9 +7,38 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ## Changes Since Last Release
 
+### Bug fixes
+
+- Fix `GOCACHE` environment variable settings when building debian source
+  package on PPA build environment.
 - Make `PS1` environment variable changeable via `%environment` section on
   definition file that used to be only changeable via `APPTAINERENV_PS1`
-  outside of container. This makes container’s prompt customizable.
+  outside of container. This makes container's prompt customizable.
+
+### Changed defaults / behaviours
+
+- When the kernel supports unprivileged overlay mounts in a user
+  namespace, the container will be constructed using an overlay
+  instead of underlay layout.
+
+### New features / functionalities
+
+- The `--no-mount` flag now accepts the value `bind-paths` to disable mounting
+  of all `bind path` entries in `apptainer.conf`.
+- Instances started by a non-root user can use `--apply-cgroups` to apply
+  resource limits. Requires cgroups v2, and delegation configured via systemd.
+- The `instance stats` command displays the resource usage every second. The
+  `--no-stream` option disables this interactive mode and shows the
+  point-in-time usage.
+- Support for `DOCKER_HOST` parsing when using `docker-daemon://`
+- `DOCKER_USERNAME` and `DOCKER_PASSWORD` supported without `APPTAINER_` prefix.
+- Add new Linux capabilities: `CAP_PERFMON`, `CAP_BPF`, `CAP_CHECKPOINT_RESTORE`.
```

Comment:
These don't belong here in the release-1.1 branch.

Reply:
I added those lines after rebase.